src/view/components/microbit/Microbit_svg.tsx

```diff
@@ -4,6 +4,8 @@
 // Adapted from : https://makecode.microbit.org/#editor
 
 import * as React from "react";
+import "../../styles/SimulatorSvg.css";

```

Comment:
Does SimulatorSvg also work for CPX? If not, maybe a different name would be clearer?

Reply:
At the moment, this css is shared amongst micro:bit and Clue. The goal is that next interns will be able to use this css for future simulators svgs as well. If we were to refactor the cpx, we should use this css as well.